core/eolearn/tests/test_aws.py

```diff
@@ -0,0 +1,71 @@
+"""
+Credits:
+Copyright (c) 2017-2019 Matej Aleksandrov, Matej Batič, Andrej Burja, Eva Erzin (Sinergise)
+Copyright (c) 2017-2019 Grega Milčinski, Matic Lubej, Devis Peresutti, Jernej Puc, Tomislav Slijepčević (Sinergise)
+Copyright (c) 2017-2019 Blaž Sovdat, Jovan Višnjić, Anže Zupanc, Lojze Žust (Sinergise)
```

Comment:
@AleksMat is this automatically generated? If not, we should add @veseln.

Reply:
It is not automatic, we should definitely add him. Also elsewhere where he contributed.

Reply:
Fixed in https://github.com/sentinel-hub/eo-learn/pull/165